logfire/_internal/config.py

```diff
@@ -514,6 +525,9 @@ class _LogfireConfigData:
     sampling: SamplingOptions
     """Sampling options."""
 
+    min_level: int
```

Comment:
Given this must be an integer I guess we should be clear that 0 (or whatever the min value we want is) means it will affect no spans

Reply:
This isn't user facing.